### ISSUE TITLE
when i click the Storage Area icon, i just get redirected to the Storage Area Explorer Webstore page

### ISSUE THREAD

**Issue by st333v** [jusio/storage-area-explorer#21]
when i click the Storage Area icon, i just get redirected to the Storage Area Explorer Webstore page

using chrome 51

**Comment by st333v** [comment]
Ah, i figured out how to use it now, you need to open the dev tools then click the "Storage Explorer" tab. The icon in the extensions bar is a bit confusing in that case...

Another problem i am having is i wish to explorer the storage area for a browser extension...

*UPDATE* Ah, i figured that out too, you need to first open the background page for the extension, then select the "Storage Explorer" tab.

Excellent extension, but could do with a little better documentation on the webstore page, that explains how it is used. A sentence or 2 on how to use it would be great for new users, thanks.

**Comment by ionutbaban** [comment]
Since there is no browserAction attached to the icon, I believe it should be removed and avoid future confusion

**Comment by jusio** [comment]
I'd like to do it, but chrome displays a browserAction for every extension regardless if there is a browserAction defined for extension (you can see the same thing for [React devtools](https://github.com/jusio/storage-area-explorer/blob/master/manifest.json)).  It can't be removed or hidden programmatically, only user has power to hide it.

**Comment by ionutbaban** [comment]
You are right the user can use the **Hide in Chrome menu** feature, but he'll still see all the hidden icons when opening the Chrome Settings Menu.

I remember reading somewhere that if you remove the browser_action block from your manifest.json the extension will not have an icon, but this changed last year and all extensions must have an icon. See more here [http://stackoverflow.com/a/37521405/3029422 ](http://stackoverflow.com/a/37521405/3029422 )

**Comment by jusio** [comment]
Yep, chrome team was worried about malicious extensions which would just seat silently in chrome and will send user data somewhere, so they forced all extensions to have an icon.
